package.json

```diff
@@ -23,6 +23,7 @@
     "backbone-computedfields": "0.0.10",
     "backbone.marionette": "^2.4.1",
     "backbone.stickit": "^0.9.2",
+    "bootstrap": "^3.3.5",
```

Comment:
We're not using the JS, just the CSS part. And we are using only some parts of it and it's all scoped under sk-container.